contracts/evoting/types/ballots_test.go

```diff
@@ -95,7 +96,7 @@ func TestBallot_Unmarshal(t *testing.T) {
 
 	// with line wrongly formatted
 	err = b.Unmarshal("x", election)
-	require.EqualError(t, err, "a line in the ballot has length != 3")
+	require.EqualError(t, err, "a line in the ballot has length != 3: x")
```

Comment:
Why add `: x` ?

Reply:
I updated the error to return the faulty line.